irida_import/main.py

```diff
@@ -41,7 +44,7 @@
                         level=logging.ERROR,
                         filemode="w")
     try:
-        config = Config()
+        config = Config(args.config)
     except FileNotFoundError:
         message = ('Error: Could not find config.ini in the irida_importer'
```

Comment:
change error message so it uses args.config

Reply:
Fixed in 5eed2b5